reference/shipping.v3.yml

```diff
@@ -397,6 +465,24 @@ components:
           readOnly: true
           format: date-time
           example: 2022-09-21T14:15:00+00:00
+    checkoutSettings:
+      title: checkoutSettings
+      description: Data about the checkout settings object.
+      type: object
+      x-internal: false
```

Comment:
```suggestion
```

I believe the x-internal flags aren't actually needed anymore (we used to use a different platform that needed them)

Reply:
Thanks, I've removed all references to `x-internal` in this file.